provisioner/docker/config_centos-7.yaml

```diff
@@ -19,6 +19,6 @@ docker:
 
 repo: "http://repos.bigtop.apache.org/releases/1.4.0/centos/7/$basearch"
 distro: centos
-components: [hdfs, yarn, mapreduce]
+components: [hdfs, yarn, mapred]
 enable_local_repo: false
 smoke_test_components: [hdfs, yarn, mapreduce]
```

Comment:
It would be nice if we can make it consistent between components and smoke_test_components (mapred vs. mapreduce). Can we change [the mapred label of the map defined in cluster.pp](https://github.com/apache/bigtop/blob/master/bigtop-deploy/puppet/manifests/cluster.pp#L57) to mapreduce?

Reply:
+1. Let's get it rolled.

Reply:
@iwasakims @evans-ye Thank you for the review. Updated cluster.pp